pkgs/by-name/op/opengv/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, eigen
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "opengv";
+  version = "0-unstable-2020-08-06";
+
+  src = fetchFromGitHub {
+    owner = "laurentkneip";
+    repo = "opengv";
+    rev = "91f4b19c73450833a40e463ad3648aae80b3a7f3";
+    hash = "sha256-LfnylJ9NCHlqjT76Tgku4NwxULJ+WDAcJQ2lDKGWSI4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    eigen
+  ];
+
+  cmakeFlakes = [
+    "-DCMAKE_BUILD_TYPE=Release"
+  ];
+
+  meta = with lib; {
```

Comment:
Consider:
- removing the `with lib;`
- sort attributes alphabetically
- add `meta.mainProgram`

Reply:
This package doesn't have an executable, so no need for a meta.mainProgram.